projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -258,7 +267,7 @@ def post_ingest_process(
         :param exceptions: Optional[List]
             A list of all exceptions (if any) encountered while ingesting data.
         """
-        pass
+        raise NotImplementedError("This method is not implemented.")
```

Comment:
Why Is this necessary?

Reply:
I was thinking that it might be useful to have some sort of warning in case one or more methods are not implemented, but it probably not necessary in this case, so I have reverted back to the use of `pass`.